.gitignore

```diff
@@ -75,3 +75,9 @@ Thumbs.db
 ## Documentation generated files
 doc/build
 doc/source/generated
+
```

Comment:
can you please explain me why you are changing the .gitignore file

Reply:
Will remove my changes here